fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/generator/OptionalContainerPropertyGenerator.java

```diff
@@ -49,15 +54,42 @@ public ContainerProperty generate(ContainerPropertyGeneratorContext context) {
 		Property property = context.getProperty();
 
 		AnnotatedType valueAnnotatedType = getValueAnnotatedType(property);
-		Property valueProperty = new ElementProperty(
-			property,
-			valueAnnotatedType,
-			0,
-			0
-		);
+		Type childType = valueAnnotatedType.getType();
+		AnnotatedType childAnnotatedType = Types.generateAnnotatedTypeWithoutAnnotation(childType);
+
```

Comment:
Is there a difference between `valueAnnotatedType` and `childAnnotatedType`?

Reply:
[0ce0cb1](https://github.com/naver/fixture-monkey/pull/951/commits/0ce0cb1e5c282bf4bd8a912b1995fb353720b1fd)
[673171a](https://github.com/naver/fixture-monkey/pull/951/commits/673171a6714f2645abfcf5cb6f44cebddc7dfb20)

I want Supplier<SimpleObject> type to property annotateType in the Supplier<Supplier<SimpleObject>> case, but I was mistaken and deleted the childAnnotateType